.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish to PyPI
+on:
+  release:
+    types: [created]
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
```

Comment:
There's a workflow that does most of the heavy lifting: https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/

Reply:
I am aware, it's too complicated. Let me see if this simple approach works.